test/test_images/performance/main.go

```diff
@@ -63,6 +64,8 @@ func init() {
 	// role=aggregator
 	flag.StringVar(&listenAddr, "listen-address", ":10000", "Network address the aggregator listens on.")
 	flag.UintVar(&expectRecords, "expect-records", 1, "Number of expected events records before aggregating data.")
+	flag.StringVar(&additionalTags, "additional-tags", "", "Array of environment specific "+
```

Comment:
Mentioning "comma separated" in the flag description would allow us to put that long example directly in the flag. I think it's more concise that way and still perfectly understandable.

Also please mention what those flags are for.